code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -209,6 +210,24 @@
 		holder.remove_reagent(/datum/reagent/consumable/capsaicin, 2)
 	..()
 
+//See block comment in ../milk/overdose_process(mob/living/M) for calculation and explanation of why this exists and why 5 was chosen
+/datum/reagent/consumable/milk/overdose_start(mob/living/M)
+	M.reagents.add_reagent(/datum/reagent/toxin/bonehurtingjuice, 5)
+	. = ..()
+
+/datum/reagent/consumable/milk/overdose_process(mob/living/M)
+	M.reagents.add_reagent(/datum/reagent/toxin/bonehurtingjuice, (0.436))
+	/*
+	* This number will not put more than 50u of BHJ into their system if only 500u(ie bare minimum OD).
+	*  milk.overdose_threshold / milk.metabolization_rate = total_cycles = 1,250 cycles
+	* (target_units / total_cycles) + BHJ.metabolization_rate = amount_to_add = .44
+	* However, livers process 1u per tick of any toxin if it is under 3u.
+	* Meaning we need a starting amount to offset this which is more than three. Ideally this should yield the lowest amount of decimal spaces, while being as low as possible.
+	* In this case starting_amount = 5.
+	* ( (target_units - starting_amount) / total_cycles) + BHJ.metabolization_rate = amount_to_add = .436
+	*/
```

Comment:
Well that's one way to do it, ty for documenting too! Ideally you'd try to calculate the rate it's being metabolized and do runtime calculations based off that instead of hardcoding, so it doesn't break with different metabolic rates and min toxin amount.

The cybernetic liver for example metabolizes out anything below 12 units and at a faster rate too iirc.

Reply:
You don't need to use parentheses for the hardcoded value either :)

Reply:
Having it able to bypass upgraded livers would be kinda shitty design, since it defeats the purpose of having to go through the upgrade process(especially .since autodoc is gone).

I also updated the parenthesis thing, added more documentation, and made the calculation more OO friendly.